### PR TITLE
Design and Create Landing Page and Internships Database

### PR DESCRIPTION
**Pull request by Ememobong28**
Designed and Created my landing page and also constructed my database for my Internships. Added some tweaks to my database.

Attached to this is a video recording of my database and a video recording TechHive Landing Page.
**Internships Database**
[Internships Database.zip](https://github.com/Ememobong28/Capstone-Project-TechHive/files/11974675/Internships.Database.zip)
**TechHive Landing Page**
[TechHive.zip](https://github.com/Ememobong28/Capstone-Project-TechHive/files/11974682/TechHive.zip)

### CODE REVIEW

File: techhive-ui/src/components/WelcomeSection/WelcomeSection.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  height: 500px;
+  /* Add additional styling for the image */
+}
+
+.text-container {
+  text-align: center;
+}
+
+.text-container h1 {
+  font-size: 48px; /* Increase the font size */
```
Comment (nicolec2626) [inline techhive-ui/src/components/WelcomeSection/WelcomeSection.css:30]: Can we remove these instructional comments from the CSS files?

File: techhive/app.js [hunk 2]
```diff
@@ -51,13 +51,16 @@ app.get('/posts', async (req, res) => {
 
 app.get('/internships', async (req, res) => {
   try {
-    const internships = await Internship.findAll();
+    const data = fs.readFileSync(path.resolve(__dirname, './internships.json'), 'utf8');
+    const { internships } = JSON.parse(data);
+    console.log(internships);
```
Comment (nicolec2626) [inline techhive/app.js:56]: typically console.log statements are used just for testing/debugging purposes. these can be removed before merging code.

**Review by nicolec2626 — APPROVED** [review]
approving, but reminder that reviews in the future are not required!

File: techhive-ui/src/components/WelcomeSection/WelcomeSection.css (hunk 1 above)
Comment (Ememobong28) [reply techhive-ui/src/components/WelcomeSection/WelcomeSection.css:30]: Okay I will, I took them out already

File: techhive/app.js (hunk 2 above)
Comment (Ememobong28) [reply techhive/app.js:56]: Okay I will take it out. Noted thank you

**Pull request merged**